### PR TITLE
Add featured projects and ecosystem section

### LINKED ISSUE #213: Projects using gtk-rs: Add carousel/slideshow
- Select 3 to 4 apps that are featured with a screenshot (and maybe app icon.)
- Maybe keep the huge list of apps for fun and for broader inspiration. Possibly hidden with some mechanisms that reveals all of them.

### PR DESCRIPTION
**Pull request by sophie-h**
- Featured Projects: Based on the "slick" carousel. This is also used
  by gtk.org.
- Ecosystem: Link to gstreamer-rs, libhandy-rs, libadwaita-rs.

##### Todo

- [ ] Agree on featured apps
- [ ] Better screenshots

Closes #213, #208

### CODE REVIEW

**Comment by sophie-h** [comment]
image

File: index.md [hunk 1]
```diff
@@ -39,6 +41,26 @@ Julian Hofer is writing a book titled [GUI development with Rust and GTK&nbsp;4]
...
+## The gtk-rs ecosystem
+
+By now, a number of additional bindings for GObject based libraries exist. While not part of the gtk-rs project, many of them are developed in close collaboration. Notable other projects within the gtk-rs ecosystem are
+
+- [**gstreamer-rs** Open source multimedia framework](https://gitlab.freedesktop.org/gstreamer/gstreamer-rs)
+- [**libhandy-rs** Building blocks for modern adaptive GNOME applications (GTK&nbsp;3)](https://gitlab.gnome.org/World/Rust/libhandy-rs)
+- [**libadwaita-rs** Building blocks for modern GNOME applications (GTK&nbsp;4)](https://gitlab.gnome.org/World/Rust/libadwaita-rs)
+{: class="box-list"}
+
+More bindings can for example be found as part of the [GNOME GitLab Rust Group](https://gitlab.gnome.org/World/Rust).
```
Comment (GuillaumeGomez) [inline index.md:54]:
```suggestion
More bindings can be found as part of the [GNOME GitLab Rust Group](https://gitlab.gnome.org/World/Rust).
```

File: js/load-carousel.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+---
```
Comment (GuillaumeGomez) [inline js/load-carousel.js:2]: Is that normal for a JS file? O.o

File: js/load-carousel.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+$(document).ready(() => {
+  let elem = $('.featured-apps');
+  if (elem) {
+    elem.slick({
+      autoplay: true,
```
Comment (GuillaumeGomez) [inline js/load-carousel.js:7]: I'm pretty sure all this can be done with CSS3 animations, no?

**Comment by GuillaumeGomez** [comment]
Overall, that seems like a lot of adds for just a slider... Can't we just have something simpler? :-/

File: js/load-carousel.js (hunk 2 above)
Comment (sophie-h) [reply js/load-carousel.js:2]: No, that activates the template engine. See the `{{ site.data.projects | where: "featured",true | size }}` line.

File: js/load-carousel.js (hunk 3 above)
Comment (sophie-h) [reply js/load-carousel.js:7]: For the animations, yes. Slick uses CSS3 for animations internally. But I don't think there is a chance for changing slides and especially swipe support without js.

**Comment by sophie-h** [comment]
If you remove swipe support it's probably not too hard to code it in js manually. But I wouldn't do it if we have a better solution with less work.

**Comment by GuillaumeGomez** [comment]
Do you mind if I give it a try? I finish something else first then I'll check what I can do.

**Comment by sophie-h** [comment]
> Do you mind if I give it a try?

Not at all!! :)

**Comment by sophie-h** [comment]
@GuillaumeGomez what's you time schedule on this? Thinking about splitting the unrelated changes into another PR.

**Comment by GuillaumeGomez** [comment]
Ah yes, please do so I don't block all the changes.

File: _layouts/default.html [hunk 4]
```diff
@@ -15,6 +15,10 @@
 
     {% include layout_footer.html %}
 
+    <script type="text/javascript" src="/js/jquery-1.11.0.min.js"></script>
```
Comment (GuillaumeGomez) [inline _layouts/default.html:18]: Didn't notice before but they're now included in most pages whereas they're not used. (Going through the stuff hehe)

Comment (sophie-h) [reply _layouts/default.html:18]: It's a dependency of the slider

**Comment by sophie-h** [comment]
Superseded by #234

**Pull request closed without merging**